docs/tools/RSDK/common/files.md

```diff
@@ -0,0 +1,24 @@
+---
+description: File format tools for Retro Engine Sonic Games
+---
+# Model Tools
+
+## ModelManiac
+Importer and exporter for RSDK model formats.
+
+[GitHub Repository :fontawesome-brands-github:](https://github.com/Rubberduckycooly/ModelManiac){ .md-button target="_blank"}
+
+# File Format Tools
+
+## RetroED
+a general purpose editor for all RSDK versions below RSDKv5.
+
+[GitHub Repository :fontawesome-brands-github:](https://github.com/Rubberduckycooly/RetroED){ .md-button target="_blank"}
+
+# Animation Tools
+
+## RSDK
+RSDK reverse stuff + Sonic Mania Animation Editor.
+
+[Download :fontawesome-solid-download:](https://github.com/Xeeynamo/RSDK/releases){ .md-button .md-button--primary target="_blank"}
+[GitHub Repository :fontawesome-brands-github:](https://github.com/Xeeynamo/RSDK){ .md-button target="_blank"}
```

Comment:
These should be split into different files. So, Model, File Format and Animation tools. If these file formats have a specific file type, then you can just create archives for said file types.